pkg/config/config.go

```diff
@@ -52,15 +54,49 @@ const (
 
 	prefixScorerBlockSizeEnvKey  = "PREFIX_SCORER_BLOCK_SIZE"
 	prefixScorerBlockSizeDefault = 256
+
```

Comment:
At this point the external process plugin is for demo only, so I would refrain from introducing and using all these environment variables (which would be eliminated by the config changes anyway).

Reply:
For a demo, you can just enable (or not) a fixed type plugins (e.g., filter or scorer) at a well known address (e.g., `localhost:8000/8088` and run their container(s) in the same Pod)